bgpd/bgp_vty.c

```diff
@@ -17256,10 +17257,14 @@ static int bgp_show_one_peer_group(struct vty *vty, struct peer_group *group,
 					dynamic ? "(dynamic)" : "",
 					peer_status);
 			}
+			member_count++;
 		}
-		if (json)
+		if (json) {
+			json_object_int_add(json_peer_group_members,
```

Comment:
this seems similar to the other recent json change proposal - to add a new object that's ... not like the existing objects in the collection.